pkgs/development/hare-third-party/hare-toml/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv
+, hare
+, scdoc
+, lib
+, fetchFromGitea
+, fetchpatch
+, nix-update-script
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "hare-toml";
+  version = "0.1.0";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "lunacb";
+    repo = "hare-toml";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-JKK5CcDmAW7FH7AzFwgsr9i13eRSXDUokWfZix7f4yY=";
+  };
+
+  patches = [
+    # Remove `abort()` calls from never returning expressions.
+    (fetchpatch {
+      name = "remove-abort-from-never-returning-expressions.patch";
+      url = "https://codeberg.org/lunacb/hare-toml/commit/f26e7cdfdccd2e82c9fce7e9fca8644b825b40f1.patch";
+      hash = "sha256-DFbrxiaV4lQlFmMzo5GbMubIQ4hU3lXgsJqoyeFWf2g=";
+    })
+  ];
+
+  nativeBuildInputs = [
+    scdoc
+    hare
+  ];
+
+  makeFlags = [
+    "HARECACHE=.harecache"
+    "PREFIX=${builtins.placeholder "out"}"
+  ];
+
+  checkTarget = "check_local";
+
+  doCheck = true;
+
+  dontConfigure = true;
```

Comment:
We shouldn't find any configure script, do we?

Reply:
No, we don't. `Hare` libraries are, usually, installed by just copying the source files onto `$out/src/hare/third-party`.

Reply:
Does it need to be $out/src ? That's not a very standard location for built packages in nixpkgs.

Reply:
It does, it's `Hare` specific thing: It looks for its third party libraries on `<prefix>/src/hare/third-party` --- see [here](https://git.sr.ht/~sircmpwn/hare/tree/546650055c9ee84b467ebafadee45c36322b1e12/item/configs/linux.mk#L29) for the related make variable embedded on the binary.